src/adaXT/criteria/criteria.pxd

```diff
@@ -48,8 +48,7 @@ cdef class Criteria:
         split index and the closest neighbour outside.
```

Comment:
Should we add the loss function here?

Reply:
No, the pxd files are only for the c defined functions. As the loss functions is a python function only (note the def), it can not and should not be defined in the pxd file. pxd files are Cythons way of creating .h files in c.